virtManager/delete.py

```diff
@@ -508,10 +508,14 @@ def _get_title_text(self, devs):
             text = title
         return [title, text]
 
-    def _vm_active_status(self)
+    def _vm_active_status(self):
         vm_active = self.vm.is_active()
         return vm_active
- 
+
+    def _delete_vm(self, vm, undefine):
```

Comment:
I don't think this function needs to take the 'undefine' value. In the caller, `undefine = vm.is_transient()` which you can check here easily enough.